.github/workflows/push.yml

```diff
@@ -59,3 +59,9 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY_ID}}
           AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
           ALB_CERTIFICATE_ARN: ${{secrets.ALB_CERTIFICATE_ARN}}
+      
+      -name: Publish NPM
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: npx lerna publish from-git 
```

Comment:
we are going to need a `--yes` added to this `run` 

https://github.com/lerna/lerna/tree/master/commands/publish#--yes

Reply:
Just at the end? Done